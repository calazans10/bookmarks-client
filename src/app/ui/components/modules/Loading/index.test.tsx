import React from 'react';
import { render } from '@testing-library/react';
import { Loading } from './';

describe('Loading', () => {
  describe('when is visible', () => {
    it('renders without crashing', () => {
      const { container } = render(<Loading isVisible />);
      expect(container.firstChild).toMatchSnapshot();
    });
  });

  describe('when is not visible', () => {
    it('renders nothing', () => {
      const { container } = render(<Loading isVisible={false} />);
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
