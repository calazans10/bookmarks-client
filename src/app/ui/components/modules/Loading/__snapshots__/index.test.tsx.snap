// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading when is not visible renders nothing 1`] = `null`;

exports[`Loading when is visible renders without crashing 1`] = `
<div
  class="sc-AxjAm foMttB"
>
  <p>
    Loading...
  </p>
</div>
`;
